### PR TITLE
Fix some issues and support to center popup multi screen for iOS < 10 

### PR DESCRIPTION
**Pull request by Ksm125**
followanalytics/fa-issues#6749
followanalytics/fa-issues#6750

### CODE REVIEW

**Comment by cjosepha** [comment]
When does this will finally be available ton prod?

**Comment by jjoaquim** [comment]
@cjosepha as soon as this is merged you can test the templates using one of the test apps.

File: multi-screen/css/style.scss [hunk 1]
```diff
@@ -16,39 +16,37 @@ body#popupTemplate.backdrop {
...
     opacity: 1;
+    position: absolute;
+    top: 50%;
+    right: 1em;
+    left: 1em;
     transition: opacity 0.05s linear, transform 0.15s ease-in-out;
 
     &--current {
-      transform: translateX(0);
+      transform: translateY(-50%);
```
Comment (firedeathz) [inline multi-screen/css/style.scss]: Why did it change from the X axis to the Y axis? Do the pages scroll vertically now somehow?

File: multi-screen/css/style.scss [hunk 2]
```diff
@@ -16,39 +16,37 @@ body#popupTemplate.backdrop {
...
 
     &--previous,
     &--next {
-      display: none;
+      opacity: 0;
     }
 
     &--previous {
-      transform: translateX(calc(-100% - 2em));
+      transform: translate(calc(-100% - 2em), -50%);
```
Comment (firedeathz) [inline multi-screen/css/style.scss]: Based on the comment above, not sure if this makes sense yet.

File: multi-screen/css/style.scss (hunk 1 above)
Comment (Ksm125) [reply multi-screen/css/style.scss]: the `absolute` property does not default centered verticaly on old OS version, `Tested on iOS < 9 ` . So i had to add `top: 50%` and then `translateY(-50%)` to get it centered.

File: multi-screen/css/style.scss (hunk 2 above)
Comment (Ksm125) [reply multi-screen/css/style.scss]: same as above

File: multi-screen/css/style.scss (hunk 1 above)
Comment (Ksm125) [reply multi-screen/css/style.scss]: didn't find other way to make it center properly

**Pull request merged**